blocks/library/text/index.js

```diff
@@ -47,7 +47,25 @@ registerBlock( 'core/text', {
 		}
 	],
 
-	edit( { attributes, setAttributes, insertBlockAfter, focus, setFocus } ) {
+	formatting: [
```

Comment:
I agree with this. I don't think a plugin should be able to change these (would love to see examples where they should). From earlier conversations I thought we wanted it like this.